### PR TITLE
Todolist

### PR DESCRIPTION
**Pull request by JHarde**
My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: components/FormInput.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <View>
+      <FormText
+        placeholder="New todo..."
+        onChangeText={(value) => changeHandler(value)}
+        value={text}
+      />
+      <Button
+        onPress={() => {
+          submitHandler(text);
+          setText("");
```
Comment (EmOlsz) [inline components/FormInput.js:28]: If we need to pass more than one line of code to event handler, then it's clear indicator for us to create separate function

File: components/Weather.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+const Weather = () => {
+  const [weatherInfo, setWeatherInfo] = useState({ name: "loading" });
+  useEffect(() => {
+    fetch(
+      "https://api.openweathermap.org/data/2.5/weather?q=stockholm&units=metric&appid=e74df95bd073adf9306ac7f46ad51144"
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        setWeatherInfo(data);
+        console.log(data);
```
Comment (EmOlsz) [inline components/Weather.js:21]: Let's remove all console.logs from project

File: components/Weather.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      });
+  }, []);
+  console.log(weatherInfo);
```
Comment (EmOlsz) [inline components/Weather.js:24]: Over here as well

**Review by EmOlsz — CHANGES_REQUESTED** [review]

**Comment by JHarde** [comment]
I have implemented most of the changes now! I think I need some more time to try to figure out how to solve the change in FormInput though!

**Comment by EmOlsz** [comment]
Great, good to hear that Joel! I think you might forgot to push your changes though, I cannot see any new commits since my review was done ;)

**Comment by JHarde** [comment]
Ah of course, haha, fixing it now! :)

**Comment by JHarde** [comment]
Taken care of now! There are some other things I'm going to try to
implement in this app during the Christmas break as well!

Den mån 14 dec. 2020 kl 17:43 skrev Joel Harde <joel.harde@gmail.com>:

**Comment by EmOlsz** [comment]
Ok, noted down! I will wait for the last set of changes, and then I will approve whole project

**Comment by JHarde** [comment]
Think I've managed to fix all the changes now! :)

**Comment by EmOlsz** [comment]
Exactly, great job :) Project approved!

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**